clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -1617,6 +1617,10 @@ void DarwinClang::AddLinkRuntimeLibArgs(const ArgList &Args,
     }
   }
 
+  if (Sanitize.needsMemProfRt())
+    if (hasExportSymbolDirective(Args))
+      addExportedSymbol(CmdArgs, "___memprof_default_options_str");
```

Comment:
@teresajohnson I know you asked for this to be refactored into a constexpr string in a header, but I don't think `Darwin.cpp` and `MemProfiler.cpp` share any headers in common, so I don't know where I would put it. I don't think I can add `MemProfiler.h` to `Darwin.cpp` because it doesn't link in the `Instrumentation` component

Reply:
Looks like this file already includes a header from llvm/ProfileData, so maybe put it into MemProf.h in that directory?

I just noticed that the symbol name here has an extra leading underscore (noticed because I couldn't find it searching the code). Which would be avoided by using a common variable. But also makes me wonder how this is working?

Reply:
Yes, Darwin symbols are prefixed with `_`, so this is correct.

Reply:
@teresajohnson is this ok? I feel like it may be more trouble than it's worth to refactor out this string